numpy/testing/_private/utils.py

```diff
@@ -20,7 +20,7 @@
 import pprint
 
 from numpy.core import(
-     intp, float32, empty, arange, array_repr, ndarray, isnat, array)
+     intp, float32, empty, arange, array_repr, ndarray, isnat, array, max)
```

Comment:
This is risky - there are places in this code that are using the builtin `max`, and you just broken them - they are not drop in replacements for each other. I'd prefer to just spell `np.max` explicitly in the handful of places where we want that one.